nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const router = express.Router();
+const reviews = require('./../data/reviews');
+
+//Respond with the json for all the reviews
+router.get('/', async(request, response) => {
+    try {
+        response.json(reviews);
+    } catch (error) {
+        console.log(error);
+        throw error;
+    }
+});
+
+//Respond with the json for the meal with the corresponding id
+router.get('/:id', async(request, response) => {
+    try {
+        const findReviewsByID = reviews.find(
+            (review) => review.id === Number(request.params.id),
+        );
+        response.json(findReviewsByID);
+    } catch (error) {
+        // throw `${}`;
+        response.end(error); // not working please help
```

Comment:
I will just have a ``throw {error}`` block here  so you'll have something like 

``
const doSomething1 = () => {
  //...
  try {
    //...
  } catch (error) {

    throw new Error(error)
  }
}
``

 here's a little article on [error handling in node.js](https://nodejs.dev/learn/error-handling-in-nodejs) .
 
 Let me know if it helps or otherwise.